src/routes/search/search-page.scss

```diff
@@ -0,0 +1,44 @@
+@media (min-width: 40em) {
+  .search-results .usa-pagination__previous-page {
+    margin-right: 120px;
+  }
+  .search-results .usa-pagination__next-page {
+    margin-left: 120px;
+  }
+}
+
+.usa-pagination__item.usa-current,
+.usa-current .usa-pagination__button,
+.usa-current .usa-pagination__button:focus {
+  border-radius: 0.25rem;
+  color: white;
+}
+
+.search-results li {
+  list-style-type: none;
+}
+
+.ais-Hits-list {
+  margin: 0;
+  padding: 0;
+}
+
+@media (min-width: 30em) {
+  .search-results .usa-input {
+    border-radius: 5px 0 0 5px;
+  }
+  .search-results .usa-search__submit-icon {
+    display: inline-block;
+    fill: white;
+  }
+}
```

Comment:
Not sure if this block is working as expected. On mobile the icon turns grey, and I feel like the border radius should be consistent on all screen sizes.

Reply:
Yeah I can't remember what the motivation for the media query was, but I don't think it has a purpose. Keeping this styles but removing the query.